code/game/objects/items.dm

```diff
@@ -632,3 +640,34 @@ GLOBAL_VAR_INIT(rpg_loot_items, FALSE)
 		M.dirty++
 
 /obj/item/proc/on_mob_death(mob/living/L, gibbed)
+
+/obj/item/proc/update_force_string()
+	if(force)
+		switch(force)
+			if(1 to 3)
+				force_string = "very low"
+			if(3 to 6)
+				force_string = "low"
+			if(7 to 9)
+				force_string = "medium"
+			if(10 to 14)
+				force_string = "<font color=green>high</font>"
+			if(15 to 20)
+				force_string = "<font color=red>robust</combat>"
+			if(20 to 25)
+				force_string = "<font color=orange>very robust</font>"

```

Comment:
sorry, I was being a bit indirect: toolboxes have default values including:
 * `force = 12`
 * `desc = "Danger. Very robust."`

It strikes me as weird to designate any range as 'very robust' except the one that includes toolboxes.